gitgud/__main__.py

```diff
@@ -155,7 +155,10 @@ def handle_init(self, args):
                 hook_file.write('#!/bin/sh' + os.linesep)
                 hook_file.write('cat - | '
                                 + python_exec + ' -m gitgud.hooks.' + module_hook_name + ' "$@"' + os.linesep)
-                hook_file.write('exit 0' + os.linesep)
+                if git_hook_name == "pre-commit":
+                    hook_file.write('exit 1' + os.linesep)
+                else:
+                    hook_file.write('exit 0' + os.linesep)
```

Comment:
This is not right.

The idea is that we can use our python hook (`precommit.py`) to give the exit code, then we don't need to clutter `handle_init`. Just add `exit(1)` to `precommit.py`

Reply:
Ah, I understand now. This was written before I knew about #128. I will add the correct exit code but be sure to merge sahansk2's pull request—It will not work until that code has been integrated.